### PR TITLE
ST-601 Upload order from your computer

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-601

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by osheaf01 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java [hunk 1]
```diff
@@ -369,6 +369,12 @@ public class CicCase {
     private CaseDocumentsCIC reinstateDocuments;
     private YesOrNo selectedCheckBox;
 
+    @CCD(
+        label = "",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private CaseDocumentsCIC orderFile;
+
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java:377]: Do we need this in CicCase.java file. Think we are using orderFile attribute defined in SendOrder.java class.

**Review by sjamihmcts — APPROVED** [review]

**Review by muradalamv1 — COMMENTED** [review]
I believe , we need to wait for mine to merge first so it will reduce conflicts.

**Review by osheaf01 — APPROVED** [review]
This LGTM now

**Pull request merged**